draft-ietf-masque-connect-ip.md

```diff
@@ -367,7 +367,9 @@ requirements:
   Capsule Protocol; see {{Section 3.2 of HTTP-DGRAM}}.
 
 If any of these requirements are not met, the client MUST treat this proxying
-attempt as failed and abort the request.
+attempt as failed and abort the request. As an example, any status code in the
+3xx range will be treated as a failure and cause the client to abort the
```

Comment:
Seems a bit odd to have 3xx called out specifically, and not 4xx or 5xx...

Reply:
It’s the most ambiguous one; some might think that a redirect would be followed. Might be good to confirm this on list if not discussed explicitly before. 

Also, is it worth pointing out that the Proxy-* fields are NOT targeted at an ip proxy?

Reply:
Not following redirects matches CONNECT-UDP / RFC 9298. Honestly I'd rather stay out of the "should you use Proxy-Authorization or Authorization header" question like we did for 9298 - it doesn't make sense to add that kind of guidance this late in the game.